app/views/shared/ref/_guides.html.erb

```diff
@@ -1,6 +1,7 @@
 <h3 class='guides'>Guides</h3>
 <ul class='unstyled'>
   <li><%= man('gitattributes') %></li>
+  <li><%= man('gitcli', 'Command-line interface') %></li>
```

Comment:
Should we add a more suggestive title? `Command-line interface conventions`?

Reply:
I think "Command-line interface conventions" is also good.